packages/webpack/src/config/base.js

```diff
@@ -6,6 +6,7 @@ import cloneDeep from 'lodash/cloneDeep'
 import escapeRegExp from 'lodash/escapeRegExp'
 import VueLoader from 'vue-loader'
 import ExtractCssChunks from 'extract-css-chunks-webpack-plugin'
+import HardSource from 'hard-source-webpack-plugin'
```

Comment:
For naming convention, i suggest importing as `HardSourcePlugin`

Reply:
Can do that, though we did not do the same iwth with `ExtractCssChunks`.

Reply:
But we do it for `TerserWebpackPlugin` :D Feel free using whatever you think is better. The point with convention is consistency.

Reply:
Also most of the `client.js` imports are in `*Plugin` form: https://github.com/nuxt/nuxt.js/blob/c3db15683a349d2a2d43d9c9965c925b401df764/packages/webpack/src/config/client.js#L8